iron-form.html

```diff
@@ -456,6 +457,20 @@
 
         this.fire('iron-form-reset');
       }, 1);
+    },
+
+    _isChildOfRegisteredParent: function(node) {
```

Comment:
what about `_isCustomElementChild`? Also, do we want this to be private or protected? 
Some docs please :)